clients/fides-js/src/components/ConsentContent.tsx

```diff
@@ -1,5 +1,5 @@
 import { ComponentChildren, VNode, h } from "preact";
-import { HTMLAttributes } from "react";
+import type { HTMLAttributes } from "react";
```

Comment:
I realized I was mistakenly importing from "react" in another place—I'm not sure if this was contributing too, but ultimately we don't really want to import from "react" because that causes "preact/compat" to kick in, which adds 2kb! I'm pretty sure importing just the `type` makes it safer